src/pages/Homepage/components/CustomersFeedback.jsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Box,
+  Container,
+  Flex,
+  Heading,
+  Text,
+  Image
+} from '@chakra-ui/react';
+import { ReactElement } from 'react';
+
+import customer_1 from '../../../images/customer-1.png';
+import customer_2 from '../../../images/customer-2.png';
+import customer_3 from '../../../images/customer-3.png';
+
+const Card = ({ name, description, logo, date }) => {
+  return (
+    <Box
+      maxW={"full"}
```

Comment:
If property is a string, no need to wrap it into { }
Just write the string itself `maxW="full"`